apps/admin-api/src/course/course.enum.graphql

```diff
@@ -0,0 +1,63 @@
+enum DayOfWeek {
```

Comment:
not sure this should this import from `codegen`?

Reply:
If you faced "cannot import local packages" problem. It's because of webpack configuration. There's 2 ways to solve the problem.

1. Fix Webpack configuration.
2. Don't use Webpack, use `Vite` + `SWC` bundler instead. (I just recently did this)

Reply:
i might migrate *some* graphql files from `api` over to `codegen`, probably only (course, course enum, override, review). Other modules are rather weird to move to codegen package (user, computation)

Reply:
and then have an npm script to copy these graphql files over before building